cmd/models/onebuild-execution-plan.go

```diff
@@ -90,3 +99,27 @@ func longestPhaseAndCommandValue(executionPlan *OneBuildExecutionPlan) (string,
 func dashesOfLength(text string) string {
 	return strings.Repeat("-", len(text))
 }
+
+// PrintBanner prints the CommandContext's name in a 72-character banner
+func (c *CommandContext) PrintBanner() {
+	centreLength := len(c.Name) + len(bannerOpen) + len(bannerClose)
```

Comment:
May be use `rune-width` instead. `len` does not necessarily give the character length that fits 72 blocks on terminal screen.

Reply:
@akhilbojedla I did not really get the idea behind using the library to get the length of a string.

Could you elaborate this little more - so that from next time onwards we can make it guideline/utility to use the same library for such use cases?

Reply:
@akhilbojedla Good call, making the change.

@gopinath-langote Golang doesn't have `chars` - it has `runes`. These `runes` represent unicode code points, whose length does not necessarily correlate to how much space they take on the terminal. See the following as an example: https://play.golang.org/p/h8DAvtrywl5

Reply:
@gopinath-langote length just gives the count of characters. When presenting something width matters. Some Unicode characters are fullwidth and use double the normal width. 

For example the width of `a` is 1 while width of `古` is 2. 

In the above example if we calculate length both of them will be 1. But since we are dealing with console output width matters. So its better to use the `rune-width` library so that the text output is more consistent.

For more information you can read this article: https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms

Reply:
@akhilbojedla So I did some testing and found that @mattn's library isn't sufficient for our needs. We need the literal number of runes in a string, not how many spaces it should occupy.

See the following example, where the first test was done with `len`, the second was done with `runewidth.StringWidth`, and the third was done with `utf8.RuneCountInString`: https://imgur.com/a/at3aD87

Reply:
@ian-howell Cool. I found about this issue with @mattn's library just couple of days ago but somehow it didn't strike when I was writing the comment. Please go ahead with using the core `utf8` library. 

The only problem I see is that when we have non-utf8 encoded strings.

Reply:
done